python/cvmfs/_common.py

```diff
@@ -75,17 +75,17 @@ def _close(self):
 
 
 
-class DatabaseObject(CompressedObject):
+class DatabaseObject:
```

Comment:
Please get rid of the `CompressedObject` class altogether. Have a look into the `History` class as well, since it works very similar to the `Catalog`.